extension.js

```diff
@@ -145,25 +130,28 @@ const main = state => {
     });
     // Set state to the previous (w/ possibly updated) values:
     state.update(STATE_VALUES_KEY, updatedStateValues);
-
-    // Flip all relevant settings BACK to stored values:
-    Object.keys(settingsToToggle).forEach(async key => {
-      // config.update(key, updatedStateValues[key])
-      const resultObj = await updateMostGranularSetting(
-        config,
-        key,
-        updatedStateValues[key]
```

Comment:
This difference in values seems to be preventing this working correctly. 
The values used should change, depending on if we loaded them from previous state, or from the extension config.